### PR TITLE
Add github actions

### PR DESCRIPTION
**Pull request by rmi22186**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
-    "shelljs": "^0.8.3",
-    "should": "^7.1.0",
-    "watchify": "^3.9.0"
-  },
-  "dependencies": {
-    "isobject": "^4.0.0",
-    "@mparticle/web-sdk": "^2.11.1"
-  },
-  "license": "Apache-2.0"
+    "name": "@mparticle/web-optimizely-kit",
```
Comment (alexs-mparticle) [inline package.json:2]: Should this be part of a separate pr?

Comment (rmi22186) [reply package.json:2]: looks like it was being prettified.  i'm okay with it being on here
